<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Event::services::qx::QxLateRegistrationsWidget</class>
 <widget class="QWidget" name="Event::services::qx::QxLateRegistrationsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblErrorMsg">
     <property name="styleSheet">
      <string notr="true">background:salmon;color:black</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="qf::qmlwidgets::TableViewToolBar" name="toolbar" native="true"/>
   </item>
   <item>
    <widget class="qf::qmlwidgets::TableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qf::qmlwidgets::TableView</class>
   <extends>QTableView</extends>
   <header location="global">qf/qmlwidgets/tableview.h</header>
  </customwidget>
  <customwidget>
   <class>qf::qmlwidgets::TableViewToolBar</class>
   <extends>QWidget</extends>
   <header location="global">qf/qmlwidgets/tableviewtoolbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
